modules/installation-special-config-storage.adoc

```diff
@@ -11,37 +11,51 @@ During an {product-title} installation, you can enable boot disk encryption and
+You can enable encryption for the boot disks on the control plane and compute nodes at installation time.
+{product-title} supports the Trusted Platform Module (TPM) v2 and Tang encryption modes.
 
-* TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor contained within a server. You can use this mode to prevent the boot disk data on a cluster node from being decrypted if the disk is removed from the server.
-* Tang: Tang and Clevis are server and client components that enable network-bound disk encryption (NBDE). You can bind the boot disk data on your cluster nodes to one or more Tang servers. This prevents the data from being decrypted unless the nodes are on a secure network where the Tang servers can be accessed. Clevis is an automated decryption framework that is used to implement the decryption on the client side.
+* TPM v2: This is the preferred mode.
+TPM v2 stores passphrases in a secure cryptoprocessor contained within a server.
+You can use this mode to prevent decryption of the boot disk data on a cluster node if the disk is removed from the server.
+* Tang: Tang and Clevis are server and client components that enable network-bound disk encryption (NBDE).
+You can bind the boot disk data on your cluster nodes to one or more Tang servers.
+This prevents decryption of the data unless the nodes are on a secure network where the Tang servers are accessible.
+Clevis is an automated decryption framework used to implement decryption on the client side.
 
 [IMPORTANT]
 ====
 The use of the Tang encryption mode to encrypt your disks is only supported for bare metal and vSphere installations on user-provisioned infrastructure.
 ====
 
-[NOTE]
-====
-On previous versions of {op-system-first}, disk encryption was configured by specifying `/etc/clevis.json` in the Ignition config. That file is not supported in clusters created with {product-title} 4.7 or above, and disk encryption should be configured by using the following procedure.
-====
+On earlier versions of {op-system-first}, disk encryption was configured by specifying `/etc/clevis.json` in the Ignition config.
+That file is not supported in clusters created with {product-title} 4.7 or later.
+Configure disk encryption by using the following procedure.
 
 When the TPM v2 or Tang encryption modes are enabled, the {op-system} boot disks are encrypted using the LUKS2 format.
 
 This feature:
 
-* Is available for installer-provisioned infrastructure and user-provisioned infrastructure deployments
+* Is available for installer-provisioned infrastructure, user-provisioned infrastructure, and Assisted Installer deployments
+  * For Assisted installer deployments:
+    - Each cluster can only have a single encryption method, Tang or TPM
+    - Encryption can be enabled on some or all nodes
+    - There is no Tang threshold; all servers must be valid and operational
+    - Encryption applies to the installation disks only, not to the workload disks{}
```

Comment:
Why are there curly brackets after the word "disks"? If you are referring to a field in a YAML CR, please mark it up appropriately.